src/test/java/org/folio/rest/api/PatronNoticePoliciesApiTest.java

```diff
@@ -209,12 +220,30 @@ public void cannotDeleteNonexistentPatronNoticePolicyId() throws MalformedURLExc
     assertThat(response.getBody(), is(NOT_FOUND));
   }
 
+  @Test
+  public void cannotDeleteInUsePatronNoticePolicy() throws InterruptedException, MalformedURLException,
+    TimeoutException, ExecutionException {
+
+    CirculationRules circulationRules = new CirculationRules()
+      .withRulesAsText(IN_USE_NOTICE_POLICY_ID);
```

Comment:
This test seems to define the rules as just being the ID of the notice policy.

To me, this does not demonstrate an example of real usage, where the ID would be included as part of a set of rules.

Reply:
> This test seems to define the rules as just being the ID of the notice policy.
> 
> To me, this does not demonstrate an example of real usage, where the ID would be included as part of a set of rules.

replaced by real `rulesAsText` record